plugins/katello/nightly/user_guide/activation_keys/index.md

```diff
@@ -118,6 +117,26 @@ To add Host Collections to a key:
 
 ![Adding Host Collections to an Activation Key](/plugins/katello/{{ page.version }}/user_guide/activation_keys/activation_key_add_host_collections.png)
 
+## Add System Purpose details to an Activation Key
+
+To add System Purpose to a key:
+
+- navigate to: Content > Activation Keys
+- select the desired key from the list
+- There is a section for System Purpose where you can set the Service Level, Usage Type, Role and Add-ons for a system. 
```

Comment:
s/ Role and Add-ons/ Role, and Add-ons/ Oxford comma if Role and Add-ons are not pair but separate items in the list